pkg/virt-controller/services/multus_annotations.go

```diff
@@ -99,8 +99,10 @@ func newMultusAnnotationData(namespace string, interfaces []v1.Interface, networ
 
 func NonDefaultMultusNetworksIndexedByIfaceName(pod *k8sv1.Pod) map[string]networkv1.NetworkStatus {
 	indexedNetworkStatus := map[string]networkv1.NetworkStatus{}
+	if pod.Annotations == nil {
+		return indexedNetworkStatus
+	}
 	podNetworkStatus, found := pod.Annotations[networkv1.NetworkStatusAnnot]
```

Comment:
What assures that annotations do exist?
Tests for example do not initialize annotations, so they are `nil`.

Reply:
Right, what I am trying to say is the same behavior remains with or w/o this condition because looking up a value of a nil map return false (and won't panic).

Reply:
Looking up a value of a nil map will panic, as the map is not initialized.
I changed this because the tests caused it to panic.

Reply:
Check out this example https://go.dev/play/p/TLAqARqDgge
Is it represent this case correctly?

Reply:
Yes, it seems to panic only if an attempt to add to it occurs.
I always thought this to be symmetric, but I should not be surprised it is not.

Thanks for noticing.
Done.